Cauldron/Assets/Scripts/Tutorial/TutorialManager.cs

```diff
@@ -59,6 +63,21 @@ private void ViewBookTutorial()
         }
     }
 
+    private void ViewAutoCookingTutorial()
+    {
+        recipeBook.OnOpenBook -= ViewAutoCookingTutorial;
+        
+        if (PlayerPrefs.GetInt(BOOK_AUTOCOOKING_KEY, 0) == 0)
```

Comment:
Erm , if you would have used OR in the recipeBook, then you would't need this check at all, because autoCooking would only happen once.